app/models/pydantic/statistics.py

```diff
@@ -77,33 +80,41 @@ class Affine(BaseModel):
     e: float = Field(..., description="Scale factor y")
     f: float = Field(..., description="Offset y")
 
+    class Config:
+        extra = "forbid"
+
 
 class BandStats(BaseModel):
-    driver: str = Field(..., description="Driver used to create raster file.")
-    interleave: str = Field(..., description="Interleave strategy.")
-    tiled: bool = Field(..., description="Raster file is tiled or not.")
-    blockxsize: int = Field(..., description="Width of tiles or strips")
-    blockysize: int = Field(..., description="Hight of tiles or stripes.")
-    compress: str = Field(..., description="Image compression used.")
-    nodata: Optional[int] = Field(..., description="No data value.")
-    dtype: str = Field(..., description="Pixel data type.")
-    width: int = Field(..., description="Raster width.")
-    height: int = Field(..., description="Raster height.")
-    bounds: Tuple[float, float, float, float] = Field(..., description="Raster founds.")
-    transform: Affine = Field(..., description="Affine transformation.")
-    crs: str = Field(..., description="Coordinate reference system.")
-    pixel_size: Tuple[float, float] = Field(..., description="Raster pixel size.")
-    origin: Tuple[float, float] = Field(..., description="Raster origin.")
+    # driver: str = Field(..., description="Driver used to create raster file.")
+    # interleave: str = Field(..., description="Interleave strategy.")
+    # tiled: bool = Field(..., description="Raster file is tiled or not.")
+    # blockxsize: int = Field(..., description="Width of tiles or strips")
+    # blockysize: int = Field(..., description="Height of tiles or stripes.")
+    # compress: str = Field(..., description="Image compression used.")
+    # nodata: Optional[int] = Field(..., description="No data value.")
+    # dtype: str = Field(..., description="Pixel data type.")
+    # width: int = Field(..., description="Raster width.")
+    # height: int = Field(..., description="Raster height.")
+    # bounds: Tuple[float, float, float, float] = Field(..., description="Raster bounds.")
+    # transform: Affine = Field(..., description="Affine transformation.")
+    # crs: str = Field(..., description="Coordinate reference system.")
+    # pixel_size: Tuple[float, float] = Field(..., description="Raster pixel size.")
     min: float = Field(..., description="Minimum pixel value.")
     max: float = Field(..., description="Maximum pixel value.")
     mean: float = Field(..., description="Mean pixel value.")
-    std_dev: float = Field(..., description="Standard deviation of pixel values.")
-    histogram: Histogram = Field(..., description="Histogram.")
+
+    histogram: Optional[Histogram] = Field(description="Histogram.")
+
+    class Config:
+        extra = "forbid"
```

Comment:
I'm a little confused what all these extra = "forbid" attributes are for

Reply:
Putting that in a model means that you can't instantiate the model with "extra" fields. So if the model is
class Blah(BaseModel):
    foo: str
    class Config:
        extra = "forbid"
and you try to do
Blah(**{"foo": "something", "bar": "something else"})
an exception will be thrown because "bar" is not a valid field for the Blah model.
Without extra="forbid" the extra fields will be silently thrown away, which can lead to errors being missed and be really confusing when debugging.

Reply:
Done. Everything that was using BaseModel is now using DataApiBaseModel.